apps/admin_panel/assets/src/omg-page-api/AdminKeySection.js

```diff
@@ -0,0 +1,288 @@
+  }
+  i[name='Key'] {
+    margin-right: 5px;
+    color: ${props => props.theme.colors.B100};
+    padding: 8px;
+    border-radius: 6px;
+    border: 1px solid ${props => props.theme.colors.S400};
+  }
+  i[name='People'] {
+    color: inherit;
+  }
+`
+
+const InputContainer = styled.div`
+  :not(:last-child) {
+    margin-bottom: 10px;
+  }
+`
+const InputLabel = styled.div`
+  font-size: 14px;
+  color: ${props => props.theme.colors.B100};
+`
+const columnsAdminKeys = [
+  { key: 'key', title: 'ADMIN KEY' },
+  { key: 'name', title: 'NAME' },
+  { key: 'created_at', title: 'CREATED DATE' },
+  { key: 'status', title: 'STATUS' },
+  { key: 'global_role', title: 'GLOBAL ROLE' }
+]
+const enhance = compose(
+  withRouter,
+  connect(
+    null,
+    { createApiKey, createAccessKey, updateAccessKey }
+  )
+)
+class ApiKeyPage extends Component {
+  static propTypes = {
+    location: PropTypes.object,
+    updateAccessKey: PropTypes.func,
+    createAdminKeyModalOpen: PropTypes.bool,
+    query: PropTypes.object,
+    fetcher: PropTypes.func
+  }
+
+  static defaultProps = {
+    fetcher: AccessKeyFetcher
+  }
+  state = {
+    createAdminKeyModalOpen: false,
+    privateKeyModalOpen: false,
+    accessKey: '',
+    secretKey: ''
+  }
+  onRequestCloseShowPrivateKey = () => {
+    this.setState({
+      secretKey: '',
+      accessKey: '',
+      submitStatus: 'DEFAULT',
+      privateKeyModalOpen: false
+    })
+  }
+
+  onSubmitSuccess = fetch => data => {
+    fetch()
+    this.setState({
+      secretKey: data.secret_key,
+      accessKey: data.access_key,
+      privateKeyModalOpen: true
+    })
+  }
+  onClickAccessKeySwitch = ({ id, expired, fetch }) => async e => {
+    await this.props.updateAccessKey({ id, expired })
+  }
+
+  rowAdminKeyRenderer = fetch => (key, data, rows) => {
+    switch (key) {
+      case 'status':
+        return (
+          <Switch
+            open={!data}
+            onClick={this.onClickAccessKeySwitch({ id: rows.id, expired: !rows.status, fetch })}
+          />
+        )
+      case 'key':
+        return (
+          <KeyContainer>
+            <Icon name='Key' /> <span>{data}</span> <Copy data={data} />
+          </KeyContainer>
+        )
+      case 'user':
+        return (
+          <KeyContainer>
+            <Icon name='Profile' /> <span>{data}</span>
+          </KeyContainer>
+        )
+      case 'created_at':
+        return moment(data).format()
+      default:
+        return data
+    }
+  }
+
+  renderConfirmPrivateKeyModal () {
+    return (
+      <ConfirmationModal
+        open={this.state.privateKeyModalOpen}
+        onRequestClose={this.onRequestCloseShowPrivateKey}
+        onOk={this.onRequestCloseShowPrivateKey}
+        confirmText='Got it!'
+        cancel={false}
+      >
+        <ConfirmCreateKeyContainer>
+          <h4>Your key pair</h4>
+          <p>
+            Please copy and keep this pair of acesss and secret key. Secret key will use to open
```

Comment:
"Please keep a copy of these secret keys. The secret key will be used to access your encrypted information." -> maybe better wording